src/drawscene.cpp

```diff
@@ -122,18 +195,24 @@ void init_texture(video::IVideoDriver* driver, const v2u32& screensize,
 			irr::video::ECF_A8R8G8B8);
 }
 
-video::ITexture* draw_image(const v2u32 &screensize,
-		paralax_sign psign, const irr::core::matrix4 &startMatrix,
-		const irr::core::vector3df &focusPoint, bool show_hud,
-		video::IVideoDriver *driver, Camera &camera, scene::ISceneManager *smgr,
-		Hud &hud, bool draw_wield_tool, Client &client,
-		gui::IGUIEnvironment *guienv, const video::SColor &skycolor)
+video::ITexture* draw_image(const v2u32& screensize,
+		paralax_sign psign, bool show_hud,
+		video::IVideoDriver* driver, Camera& camera, scene::ISceneManager* smgr,
+		Hud& hud, bool draw_wield_tool, Client& client, gui::IGUIEnvironment* guienv,
+		video::SColor skycolor )
 {
 	static video::ITexture* images[2] = { NULL, NULL };
 	static v2u32 last_screensize = v2u32(0, 0);
+	scene::ICameraSceneNode *cameraNode = camera.getCameraNode();
 
 	video::ITexture* image = NULL;
 
+	float halfInterocularDistance = g_settings->getFloat("3d_paralax_strength");
```

Comment:
This setting `3d_paralax_strength` is documented as `3d_parallax_strength` (two `l`s)  in the minetest.conf.example file.
It should be corrected here and in the default settings aswell.